.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Report bugs or errors
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run code '...'
+2. Expect this error '....'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - R Version [e.g. 4.4.0]
+ - Python Version [e.g. 3.10]
+ - flepiMoP branch commit if relevant 
```

Comment:
I would add here the commands to get these versions (because our users are sometimes less computer savvy than other programmers:
`python --version` and `R --version`

also ask for the config file name and a link to the config repository OR the step in the tutorial that failed.

Reply:
I would also add the command to get the git hash: `git rev-parse HEAD`

Reply:
I see. That makes a lot of sense...I also had a vague concern about making issues too time-consuming to open. If you feel that it should be reduced, I could easily reduce it to just a "description" and "relevant people/other issues" or something of that nature. I was primarily going off of the suggested issue templates that GitHub provided. Please let me know if you'd like me to prune it and I can.

Reply:
I agree with this. Adding those commands will help a lot.